ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/InMemory.hs

```diff
@@ -667,9 +664,9 @@ ledgerDbCurrentValues =
 -- | The ledger state at the tip of the chain
 ledgerDbCurrent :: GetTip (l EmptyMK) => LedgerDB l -> l EmptyMK
 ledgerDbCurrent =
-    either unCheckpoint unCheckpoint
+    either unDbChangelogState unDbChangelogState
```

Comment:
Discussed with Nick: revert to dbChangelogVolatileCheckpoints so that we don't have to reach for the internals.